README.md

```diff
@@ -154,6 +155,7 @@ When you use this template, try follow the checklist to update your info properl
 
 - [ ] Change the author name in `LICENSE`
 - [ ] Change the title in `App.vue`
+- [ ] Change the hostname in `vite.config.ts`
```

Comment:
@userquin vite-plugin-pages-sitemap automatically rewrite robots.txt with this hostname